config.go

```diff
@@ -252,56 +252,55 @@ type LogBuffering struct {
+	LocalStorage   int    `bson:"local_storage_size" json:"local_storage_size" yaml:"local_storage_size"`
 }
 
 type SlackConfig struct {
-	Options *send.SlackOptions `yaml:"options"`
-	Token   string             `yaml:"token"`
-	Level   string             `yaml:"level"`
+	Options *send.SlackOptions `bson:"options" json:"options" yaml:"options"`
+	Token   string             `bson:"token" json:"-" yaml:"token"`
+	Level   string             `bson:"level" json:"level" yaml:"level"`
 }
 
 type NewRelicConfig struct {
-	ApplicationName string `yaml:"application_name"`
-	LicenseKey      string `yaml:"license_key"`
+	ApplicationName string `bson:"application_name" json:"application_name" yaml:"application_name"`
+	LicenseKey      string `bson:"license_key" json:"-" yaml:"license_key"`
 }
 
 // Settings contains all configuration settings for running Evergreen.
 type Settings struct {
-	Database            DBSettings                `yaml:"database"`
-	WriteConcern        WriteConcern              `yaml:"write_concern"`
-	ConfigDir           string                    `yaml:"configdir"`
-	ApiUrl              string                    `yaml:"api_url"`
-	AgentExecutablesDir string                    `yaml:"agentexecutablesdir"`
-	ClientBinariesDir   string                    `yaml:"client_binaries_dir"`
-	SuperUsers          []string                  `yaml:"superusers"`
-	Jira                JiraConfig                `yaml:"jira"`
-	Splunk              send.SplunkConnectionInfo `yaml:"splunk"`
-	Slack               SlackConfig               `yaml:"slack"`
-	Providers           CloudProviders            `yaml:"providers"`
-	Keys                map[string]string         `yaml:"keys"`
-	Credentials         map[string]string         `yaml:"credentials"`
-	AuthConfig          AuthConfig                `yaml:"auth"`
-	RepoTracker         RepoTrackerConfig         `yaml:"repotracker"`
-	Api                 APIConfig                 `yaml:"api"`
-	Alerts              AlertsConfig              `yaml:"alerts"`
-	Ui                  UIConfig                  `yaml:"ui"`
-	HostInit            HostInitConfig            `yaml:"hostinit"`
-	Notify              NotifyConfig              `yaml:"notify"`
-	Scheduler           SchedulerConfig           `yaml:"scheduler"`
-	Amboy               AmboyConfig               `yaml:"amboy"`
-	Expansions          map[string]string         `yaml:"expansions"`
-	Plugins             PluginConfig              `yaml:"plugins"`
-	IsNonProd           bool                      `yaml:"isnonprod"`
-	LoggerConfig        LoggerConfig              `yaml:"logger_config"`
-	LogPath             string                    `yaml:"log_path"`
-	PprofPort           string                    `yaml:"pprof_port"`
-	GithubPRCreatorOrg  string                    `yaml:"github_pr_creator_org"`
-	NewRelic            NewRelicConfig            `yaml:"new_relic"`
+	Database           DBSettings                `yaml:"database"`
```

Comment:
Would you mind alphabetizing this list? It'll make is easier to read/understand diffs to it in the future?